app/helpers/conversations_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ConversationsHelper
```

Comment:
This gets generated by rails generate controller - if you don't add to it, delete it. I think you can also pass --skip-helper to the generate command.